DeepSpeech.ipynb

```diff
@@ -1315,7 +1319,12 @@
     "    graph = tf.Graph()\n",
     "    with graph.as_default():\n",
     "        # Obtain all the data\n",
-    "        data_sets = ds_importer_module.read_data_sets(ds_dataset_path, batch_size, n_input, n_context)\n",
+    "        data_sets = ds_importer_module.read_data_sets(ds_dataset_path, \\\n",
```

Comment:
During refactoring for the last big PR, I thought about splitting the `ds_importer_module.read_data_sets` into separate calls - one per set. It could save us quite some redundancy. Now we are reading all sets for all three different set executions train/dev/test. I don't now what the exact overhead is, but at least the code would get a bit simpler.